src/main/java/dev/joopie/jambot/util/YouTubeLinkParser.java

```diff
@@ -0,0 +1,31 @@
+package dev.joopie.jambot.util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class YouTubeLinkParser {
```

Comment:
Was er een noodzaak om dit in een util class te stoppen? Ik ben eigenlijk geen fan van dit soort classes. Probeer methodes zoveel mogelijk bij classes te houden die het gebruiken. Dan wel op de entity/dto, of zoals het was in de service.

Reply:
Deleted!